nerfstudio/process_data/process_data_utils.py

```diff
@@ -184,9 +183,10 @@ def convert_video_to_images(
             + ";".join(downscale_chains)
         )
 
+        ffmpeg_cmd += " -vsync vfr"
+
         if spacing > 1:
             CONSOLE.print("Number of frames to extract:", math.ceil(num_frames / spacing))
```

Comment:
I believe there is still a bug in this PR: num_frames is still used when spacing > 1 but the variable is removed, so will result in an error.